fern/docs/pages/sdks/node-sdk.mdx

```diff
@@ -1,27 +1,28 @@
 [Source code](https://github.com/fern-api/fern-typescript)
 
-**Latest version**: `0.8.13`
+**Latest version**: `0.9.5`
 
-The TypeScript Node.js SDK generator outputs a fully functional TypeScript/JavaScript SDK for server-side usage. It can publish the SDK to npmjs.org (or any other npm repository).
+The TypeScript Node.js SDK generator outputs a fully functional TypeScript/JavaScript SDK for server-side usage. It can publish the SDK to [npmjs.org](https://npmjs.org) (or any other Node package repository).
```

Comment:
```suggestion
The TypeScript Node.js SDK generator outputs a fully functional TypeScript/JavaScript SDK for server-side use. It can publish the SDK to [npmjs.org](https://npmjs.org) (or any other Node package repository).
```